src/main/java/de/tyrannus/cleandebug/CleanDebug.java

```diff
@@ -162,6 +162,18 @@ public static void modifyRightText(List<String> text) {
             }
         }
 
+        if (CleanDebugConfig.hideFigura) {
+            var figuraIndex = indexOfStartingWith(text, "§b[Figura]");
+
+            if (figuraIndex != -1) {
+                text.subList(figuraIndex, Math.min(figuraIndex + 3, text.size())).clear();
+
+                if (figuraIndex > 0 && text.get(figuraIndex - 1).isEmpty()) {
+                    text.remove(figuraIndex - 1);
+                }
+            }
+        }
+
```

Comment:
I tested it, I believe it works properly, but it would be nice if you could review it anyway!